README.md

```diff
@@ -1,17 +1,192 @@
-## My Project
+# A Unified Efficient Pyramid Transformer for Semantic Segmentation
 
-TODO: Fill this README out!
+## Installation
 
-Be sure to:
+* Linux, CUDA>=10.0, GCC>=5.4
+* Python>=3.7
+* Create a conda environment:
 
-* Change the title in this README
-* Edit your repository description on GitHub
+```bash
+    conda create -n unept python=3.7 pip
+```
 
-## Security
+Then, activate the environment:
+```bash
+    conda activate unept
+```
+* PyTorch>=1.5.1, torchvision>=0.6.1 (following instructions [here](https://pytorch.org/))
+
+For example:
+```
+conda install pytorch==1.5.1 torchvision==0.6.1 cudatoolkit=10.2 -c pytorch
+```
+
+* Install [MMCV](https://mmcv.readthedocs.io/en/latest/), [MMSegmentation](https://github.com/open-mmlab/mmsegmentation/blob/master/docs/install.md), [timm](https://pypi.org/project/timm/)
+
+```
+pip install -r requirements.txt
+```
+
+* Install [Deformable DETR](https://github.com/fundamentalvision/Deformable-DETR) and compile the CUDA operators
```

Comment:
Need more details, like step-by-step guide. Or you can provide the readme page in Deformable DETR that teaches user how to compile the CUDA operators.